content/en/serverless/libraries_integrations/library.md

```diff
@@ -32,7 +32,8 @@ The Datadog Lambda Library is responsible for:
 - Submitting [custom metrics][2] (synchronously and asynchronously).
 - Enabling [Datadog APM and Distributed Tracing][3] for Node.js, Python, and Ruby.
 
-You **must** also install and configure the Datadog Forwarder to ingest traces, enhanced Lambda metrics, or custom metrics (asynchronously) from your Lambda functions.
+If you are using the Datadog Lambda library for **Ruby** or **Java**, you **must** also install and configure the Datadog Forwarder to ingest traces, enhanced Lambda metrics, or custom metrics (asynchronously) from your Lambda functions.
+If you are using the Datadog Lambda library for **Python**, **Node**, or **Go**, you may use the [Datadog Lambda Extension][9] to ingest traces, enhanced Lambda metrics, or custom metrics; or you may continue to use Datadog Forwarder.
```

Comment:
Should this say, "or you may continue to use the Datadog Forwarder."?

Reply:
I'm using the semicolon to separate items in a list that contains other commas. https://writing.wisc.edu/handbook/grammarpunct/semicolons/#:~:text=Use%20a%20semicolon%20between%20items,expensive%20but%20quick%20and%20neat.

Reply:
Ooooh, _the_